kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -180,17 +179,13 @@ export default {
     },
     selectedReceiver(selectedReceiver) {
       if (selectedReceiver) {
-        this.isEditReceiver = false;
-        this.receiverId = selectedReceiver.remoteId;
-        this.identity = {
-          avatar: selectedReceiver.profile.avatarUrl,
-          external: selectedReceiver.profile.external,
-          fullname: selectedReceiver.profile.fullName,
-          id: selectedReceiver.remoteId,
-          identityId: selectedReceiver.identityId,
-          isUserType: true,
-          type: 'user',
-          username: selectedReceiver.remoteId};
+        if (this.receiverId !== selectedReceiver.remoteId) {
+          this.receiverId = selectedReceiver.remoteId;
+          this.$root.$emit('kudos-notification-alert', {
```

Comment:
We have added a new common component for alert notification here :
https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/popover/components/NotificationAlerts.vue.

Reply:
Done! Thanks